release-notes/release-notes-template.md

```diff
@@ -0,0 +1,55 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud < release-version >.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Feature 1
+
+< Explain feature 1 >
+
+### Feature 2
+
+< Explain feature 2 >
+
+
+< Add feature sections as necessary >
+
+
+### Dashboard improvements
```

Comment:
I compared this with the 1.18.0 md, there are
- Streaming improvements
- Dashboard improvements

Are we flexible about the sub-sections under the `New features & improvements` or we want to have fine-grinded sub-sections for each portion in our stack?

Reply:
I think it is worth having separate stack headings in the template. When we actually write the release notes, if we don't have significant improvements to one stack, we can always remove the section. So for now, I have categorised as Core stack improvements, Streaming improvements, Dashboard improvements.